### PR TITLE
app demonstrating use of services and workers

### PR DESCRIPTION
**Pull request by nawiju**
Created a simple android app that demonstrates the use of workers and services that run in the background

### CODE REVIEW

**Review by SzymonKozl — APPROVED** [review]

**Review by GustawB — CHANGES_REQUESTED** [review]
1. Please merge changes from the main branch and move this under research. Rename it to something more meaningful.
2. .idea folder should not be a part of the project (this is IDE-specific). Same for gradle folder. This should be handled in .gitignore. For the app layout you can refer to e.g https://github.com/ggerganov/llama.cpp/tree/master/examples/llama.android; I was able to run this app without any additional installations etc.
3. This is not really what this task was about. I guess we described it in a bad way, but in DoD it is said that the aim of this task is to show an example of accesibility feature.
Those were suggested to us by Konrad in https://developer.android.com/guide/topics/ui/accessibility/service. What you did instead is just spawning a new thread, which is not really a challenge, and is definitely not a solution to our problem. I suggest we keep this PR open until the next weekly.

**Review by GustawB — CHANGES_REQUESTED** [review]
First of all, accessibility service now works and works better than I expected so good job on that.
Still, you didn't address any of my previous requests. .idea and .gradle folders are still in repo so there is a ton of unnecessary file changes. And, because these folders were created on your machine, I wasted time debugging something while the fix was to delete those folders so pleeease, .gitignore.
Also, please merge main branch into this and move your project under research. Also, rename it to something more meaningful.
Another also, but on main branch there is a .gitignore that works so maybe it will fix your issue.
Lastly, when I asked for .md file, I meant README.md inside the TestApplication1.
To run everything smoothly I run only the project folder so I had to switch between IDE and firefox to actually access instructions how to enable this app in emulator.
Lastly, .md != .pdf

File: research/ServiceDemoApp/.gitignore [hunk 1]
```diff
@@ -0,0 +1,186 @@
+# Gradle files
```
Comment (GustawB) [inline research/ServiceDemoApp/.gitignore:1]: Tried to run it locally, it seems that the .gitignore from src is too strong; please try this one

```suggestion
# Gradle files
.gradle/
build/

# Local configuration file (sdk path, etc)
local.properties

# Log/OS Files
*.log

# Android Studio generated files and folders
captures/
.externalNativeBuild/
.cxx/
*.apk
output.json

# IntelliJ
*.iml
.idea/
misc.xml
deploymentTargetDropDown.xml
render.experimental.xml

# Keystore files
*.jks
*.keystore

# Google Services (e.g. APIs or Firebase)
google-services.json

# Android Profiling
*.hprof
```

File: README.md
Comment (GustawB) [inline README.md:1]: You accidentally removed the "main" README. Please add it back (you can copy it from main branch.)

**Review by GustawB — CHANGES_REQUESTED** [review]

**Review by GustawB — CHANGES_REQUESTED** [review]
You added back the README from the source, but removed the README that was inside Android Studio. It contained tutorial how to enable your app in the emulator. You can have multiple README's in project; you can look here for reference: https://github.com/ZPP-MURMURAS/ZPP_Murmuras/tree/main/research/HuggingFaceModels.  in my research I added a simple README, while retaining the one at the root of the project

**Review by GustawB — APPROVED** [review]

**Pull request merged**